backend/src/libs/packages/config/libs/types/environment-schema.type.ts

```diff
@@ -13,6 +13,12 @@ type EnvironmentSchema = {
     POOL_MIN: number;
     POOL_MAX: number;
   };
+  JWT: {
+    SECRET_KEY: string;
+    ALG: {
+      JWT_ALG: string;
+    };
```

Comment:
Shouldn't it be like so?

```suggestion
    ALG: string;
```

Reply:
I think you are right, I attached the latest implementation [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/37#discussion_r1301414975)